Units/parser-sql.r/sql-create-table-as.d/input.sql

```diff
@@ -0,0 +1,9 @@
+--
+-- Taken from #1658 submitted by @karlb
+--
+CREATE TABLE foo(
+    col1 text
+);
+
+CREATE TABLE bar AS
+SELECT 1 AS col2;
```

Comment:
there probably should be a note somewhere that this is *not* the expected output, but just here until the parser supports the other tests.  The thing is that it can be confusing when working on the parser to have a test starting to fail, although you actually just *fixed* (or improved) something.

Reply:
I see. I will add README (or extend the parser to capture `col2`).